tools/debian/control

```diff
@@ -77,7 +77,6 @@ Package: cockpit-docker
 Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python,
```

Comment:
Nothing in the shipped package seems to:
```sh
$ dpkg-deb -x cockpit-docker_0-1_all.deb c
$ grep -r python c
# (no results)
```
Although there clearly are hits in `git grep python {dist,pkg}/docker`, but neither `cockpit-atomic-storage` nor `storage.jsx` is shipped in the binary docker package. Should they be?